src/coreclr/jit/emitxarch.cpp

```diff
@@ -11241,7 +11241,7 @@ void emitter::emitDispIns(
 
         case IF_RWR_RRD_RRD:
         {
-            assert(IsVexEncodableInstruction(ins));
+            assert(IsVexOrEvexEncodableInstruction(ins));
```

Comment:
heh, I hit it in https://github.com/dotnet/runtime/pull/84854 too 🙂 will rebase once your PR is merged